doc/students.md

```diff
@@ -392,15 +422,31 @@ None
 
```

Comment:
We have 2 endpoints: /students (PUT) for edit list of student and /students/{id} (PUT) for edit one student. I think it need to remove "{id}" in line 409 and add separately info about endpoint /students/{id} (PUT)